content/en/opentelemetry/interoperability/instrumentation_libraries.md

```diff
@@ -115,15 +115,57 @@ OpenTelemetry's [Agent Configuration](https://opentelemetry.io/docs/zero-code/ja
 
 {{% /tab %}} -->
 
-<!-- {{% tab "Go" %}}
+{{% tab "Go" %}}
 
 ## Compatibility requirements
 
+The Datadog SDK for Go supports library instrumentations written using the [Opentelemetry-Go Trace API][1]. This includes the [opentelemetry-go-contrib/instrumentation][2] libraries, however, anything that relies on metrics or logs exporters is not supported at this time.
+
 ## Getting started
 
+Set up the Datadog trace provider and configure Opentelemetry to use the Datadog trace provider as per the Imports and Setup instructions listed in the [Go Custom Instrumentation using OpenTelemetry API guide][3].
+
+Then, follow the instructions of your chosen opentelemetry-go-contrib library to instrument your service.
+
+Example:
```

Comment:
Change this to something like "The following is an example using the _____ library:"